### ISSUE TITLE
Unable to extract audio

### ISSUE THREAD

**Issue by beew** [axcore/tartube#124]
Hi, I am unable to extract audio from videos (e.g mp3)  In fact it seems that tartube is unable to download any format other than video in mp4 or webm. .

e.g tried this link https://www.youtube.com/watch?v=rLtZKkCIVmI  I don't want the video, I just want the audio. I use the classic interface and choose format mp3 from the box.

Here is the output.

```
/home/bee/.local/bin/youtube-dl --newline -i --hls-prefer-native -o /home/bee/Music/%(title)s.%(ext)s -f mp3 --ffmpeg-location /usr/bin/ffmpeg https://www.youtube.com/watch?v=rLtZKkCIVmI
[youtube] rLtZKkCIVmI: Downloading webpage
ERROR: requested format not available
```

However, using youtube-dl with the comand line downloads video, extracts audio and converst to desired format with no problem.

This works flawlessly
```
$youtube-dl -x --audio-format mp3 https://www.youtube.com/watch?v=rLtZKkCIVmI
[youtube] rLtZKkCIVmI: Downloading webpage
[download] Destination: Tom Waits - 'Downtown Train'-rLtZKkCIVmI.m4a
[download] 100% of 4.66MiB in 00:02
[ffmpeg] Correcting container in "Tom Waits - 'Downtown Train'-rLtZKkCIVmI.m4a"
[ffmpeg] Destination: Tom Waits - 'Downtown Train'-rLtZKkCIVmI.mp3
Deleting original file Tom Waits - 'Downtown Train'-rLtZKkCIVmI.m4a (pass -k to keep)

```
Tartube v2.1.051 installed with pip3

youtube-dl version 2020.06.16.1

OS Ubuntu 20.04

**Comment by axcore** [comment]
@beew, in the Classic Mode tab, there is a dropdown box called 'Format'. This box is broken, so you can't use it.

Here is a workaround. I have tested it with your video, and I can download the .mp3 without problems.

1. In the classic mode tab, click the menu icon in the top-right corner, and select 'Edit classic download options'
2. In the new window, if the 'Show advanced download options" button is broken, click it
3. Click the Post-process tab
4. Click the button Post-process video files to convert them to audio-only files to select it
5. In the dropdown box Audio format of the post-processed file, select mp3
6. Click the OK button to close the window

**Comment by beew** [comment]
@axcore 

Your workaround works but it seems that you have to edit classical download option everytime if you want to switch from audio to video or download in a different video or audio format (say instead of mp3 I now want flac) In youtube-dlg you used to be able to change format on the fly by choosing from the format drop down box.

**Comment by axcore** [comment]
You could install Tartube v2.1.0, which does not have this problem, until I can produce a fix.

**Comment by beew** [comment]
@axcore 

V2.1.0 is not installable. pip3 install tartube==2.1.0 gave

```
ERROR: Could not find a version that satisfies the requirement gi (from tartube==2.1.0) (from versions: none)
ERROR: No matching distribution found for gi (from tartube==2.1.0)
```

**Comment by axcore** [comment]
v2.1.0 could not be installed via pip (this was fixed in v2.1.051). Other installation methods for v2.1.0 (DEB, RPM, run from source code) do work.

**Comment by axcore** [comment]
Fixed in v2.1.057. The new code can be downloaded from github or pypi. I won't updated the main installers yet because I'm waiting for a youtube-dl fix.
